packages/graphql-typescript-definitions/src/cli.ts

```diff
@@ -47,31 +54,62 @@ const argv = yargs
   })
   .help().argv;
 
+if (typeof argv.cwd === 'string') {
+  process.chdir(argv.cwd);
+}
+
+// convert the graphql globs to a relative path (if provided)
+// this is to deal with an issue with graphql-config and absolute path globbing
+// see: https://github.com/stephen/vscode-graphql/issues/23
+const graphQLFiles = argv._[0] && relative(process.cwd(), argv._[0]);
```

Comment:
Here is another one that we could conceivably just do the breaking change and only allow it to be defined through the GraphQL config